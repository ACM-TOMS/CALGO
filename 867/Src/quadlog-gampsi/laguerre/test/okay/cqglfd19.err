  ERROR: qglqfd() returns ierr =          3
  Illegal value of alpha:   -2.00e+00 <= -1
  ERROR: qglqfd() returns ierr =          3
