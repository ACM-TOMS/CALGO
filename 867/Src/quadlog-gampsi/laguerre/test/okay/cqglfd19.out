  Test integral: int(x^p * x^alpha * exp(-x) * ln(x), x = 0..infinity)

  1 ulp =   1.93e-34

  Floating-point precision appears to be (1 - log2(ulp) =) 113 bits

  alpha =   1.000000  nquad =      0

  alpha =   1.000000  nquad =  32767

  Done
